src/biosHandler.cpp

```diff
@@ -0,0 +1,49 @@
+#include "biosHandler.hpp"
+
+#include "logger.hpp"
+
+namespace vpd
+{
+void BiosHandler::checkAndListenPLDMService()
+{
+    // Setup a call back match on NameOwnerChanged to determine when PLDM is up.
+    static std::shared_ptr<sdbusplus::bus::match_t> nameOwnerMatch =
+        std::make_shared<sdbusplus::bus::match_t>(
+            *m_asioConn,
+            sdbusplus::bus::match::rules::nameOwnerChanged(
+                "xyz.openbmc_project.PLDM"),
+            [this](sdbusplus::message_t& msg) {
+        if (msg.is_method_error())
+        {
+            logging::logMessage("Error in reading PLDM name owner signal.");
+            return;
+        }
+
+        std::string name;
```

Comment:
l_serviceName  and l_* prefix for all local variables ?

Reply:
Yup, missed that completely.